parity/cli/mod.rs

```diff
@@ -800,7 +800,7 @@ usage! {
 
 			ARG arg_logging: (Option<String>) = None, or |c: &Config| c.misc.as_ref()?.logging.clone(),
 			"-l, --logging=[LOGGING]",
-			"Specify the logging level. Must conform to the same format as RUST_LOG.",
+			"Specify the general logging level (error, warn, info, debug or trace). It can also be set for a specific module, example: '-l sync=debug, rpc=trace'",
```

Comment:
Does it really work with space between the options?

Reply:
```
parity -l sync=debug, rpc=trace
error: Found argument 'rpc=trace' which wasn't expected, or isn't valid in this context

USAGE:
    parity [SUBCOMMAND]

For more information try --help

```

Reply:
Damn I tested the example but didn't copy paste it so didn't catch the space.